src/server/game/Handlers/AuctionHouseHandler.cpp

```diff
@@ -621,46 +566,25 @@ void WorldSession::HandleAuctionListBidderItems(WorldPacket& recvData)
 
     AuctionHouseObject* auctionHouse = sAuctionMgr->GetAuctionsMap(creature->getFaction());
 
-    WorldPacket data(SMSG_AUCTION_LIST_BIDDER_ITEMS_RESULT, (4+4+4));
+    WorldPackets::AuctionHouse::AuctionListBidderItemsResult result;
+
     Player* player = GetPlayer();
-    data << uint32(0);                                     //add 0 as count
     uint32 count = 0;
     uint32 totalcount = 0;
-    while (outbiddedCount > 0)                             //add all data, which client requires
-    {
-        --outbiddedCount;
-        uint32 outbiddedAuctionId;
-        recvData >> outbiddedAuctionId;
-        AuctionEntry* auction = auctionHouse->GetAuction(outbiddedAuctionId);
-        if (auction && auction->BuildAuctionInfo(data))
-        {
-            ++totalcount;
-            ++count;
-        }
-    }
-
-    auctionHouse->BuildListBidderItems(data, player, count, totalcount);
-    data.put<uint32>(0, count);                           // add count to placeholder
-    data << totalcount;
-    data << uint32(300);                                  //unk 2.3.0
-    SendPacket(&data);
+    auctionHouse->BuildListBidderItems(result, player, count, totalcount);
+    // to check
+    result.TotalCount = count;
+    result.DesiredDelay = totalcount;
```

Comment:
according to my research, DesiredDelay is in sniff always 300. TotalCount is maybe = totalcount, not sure about this, because on retail its a bit strange, so if you have researched something else tell me